### PR TITLE
.coafile: Add commit checks

### PR DESCRIPTION
**Pull request by nvzard**
Add checks for the shortlog of commit

Closes https://github.com/coala/coala-quickstart/issues/38

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -28,6 +28,8 @@ bears = LineLengthBear
 
 [commit]
 bears = GitCommitBear
+shortlog_trailing_period = False
+shortlog_regex = ([^:]*|[^:]+[^ ]: [A-Z0-9*].*)
```
Comment (cornu-ammonis) [inline .coafile]: commenting explaining this regex could be helpful

**Comment by cornu-ammonis** [comment]
8fc6fb1 needs work

**Comment by nvzard** [comment]
I'm not sure if we can add comments

**Comment by cornu-ammonis** [comment]
Maybe you should specify it in the commit body ? :)

**Comment by nvzard** [comment]
I think it's not where specified in the commit guidelines.

**Comment by cornu-ammonis** [comment]
What do you mean? Commit guidelines say "Describe the reasoning of your changes
in maximum of 72 characters per line" -- explaining how your regex is supposed to work would be reasoning for your changes, no?

**Comment by RJ722** [comment]
@cornu-ammonis You need to explain the change you have done through the commit in the commit shortlog and generally, it shouldn't contain technical details. It **isn't** a replacement for inline comments.

Here, for example, a commit message could be:
```
.coafile: Add check for commit message

Check whether the commit shortlog follows the standard
guidelines
```

**Comment by nvzard** [comment]
@RJ722 thank you for explaining :)

**Comment by cornu-ammonis** [comment]
ack 8fc6fb

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**